### PR TITLE
NEW: Sticky component

### PR DESCRIPTION
**Pull request by mainframev**
This Pull Request meets the following criteria:

* [ ] Tests have been added/adjusted for my new feature
* [ ] New Components are registered in index.js of my project
<br/><br/><br/><url>LiveURL: https://orbit-components-new-FloatingCard.surge.sh</url>

### CODE REVIEW

File: src/FloatingCard/index.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+import { addScrollHandler, removeScrollHandler, getScrollingElement } from "../utils/scroll";
+import defaultTokens from "../defaultTokens";
+
+import type { Props, State } from "./index";
+
+// TODO: add orbit token for box-shadow
+const FloatingWrapper = styled.div`
+  position: ${({ sticky }) => (sticky ? `fixed` : `relative`)};
+  ${({ size }) => size.height && `top: ${size.height}px`};
```
Comment (wEprous) [inline src/FloatingCard/index.js]: Probably:
```
${({ size }) => css`
  top: ${size.height && `${size.height}px`};
  width: ${size.width && `${size.width}px`};
`};
```

File: src/utils/scroll/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// @flow
+export const addScrollHandler = (onScrollFunction: () => void, element: Node = document) => {
```
Comment (wEprous) [inline src/utils/scroll/index.js]: Shouldn't be there `index.js.flow` for these utilities? :)

**Comment by wEprous** [comment]
LGTM, @tomashapl can you look? 🙂

**Comment by tomashapl** [comment]
@mainframev Also please update `handleTop` on window resize, thanks

**Pull request merged**